### PR TITLE
Update Deta information

### PR DESCRIPTION
**Pull request by Nazar-Ant**
I was looking into hosting on Deta for my new project and noticed that its documentation describes that it supports most web frameworks and even Deno, but this is not listed on the grammY website. There was a question mark on the grammY website, though, so you probably knew about this feature but didn't check it out, right? If that's true, I can check it out on my own project to make sure.

### CODE REVIEW

**Comment by dcdunkan** [comment]
Just a note:

> There was a question mark on the grammY website, though, so you probably knew about this feature but didn't check it out, right?

I don't why it was a question mark, but I don't think that was the reason. Deta (**deta.sh**) only supported Node.js and Python apps. They recently announced that they are moving to deta.space, a new, better platform (they gave the old users time to migrate their apps - until June 1). And the new Deta (deta.space) apps currently supports more languages, as you said, including Deno.

**Comment by Nazar-Ant** [comment]
@dcdunkan I see. So eventually deta.sh will disappear altogether and be completely replaced by deta.spase?

**Comment by dcdunkan** [comment]
As far as I understand, yes. See https://deta.space/from-cloud and their announcement: https://deta.space/blog/introducing-space.

**Comment by KnorpelSenf** [comment]
What are we going to do with this?

**Comment by Nazar-Ant** [comment]
@KnorpelSenf Who are you addressing?)

**Comment by KnorpelSenf** [comment]
Anyone who has an answer to my question

**Comment by roj1512** [comment]
@KnorpelSenf I think we can close this. Correct me if I’m wrong @dcdunkan @Nazar-Ant.

**Comment by Nazar-Ant** [comment]
@roj1512 Do you think this change is unnecessary?

**Comment by roj1512** [comment]
This change is for deta.space, right?

**Comment by roj1512** [comment]
And the comparison talks about deta.sh?

**Comment by dcdunkan** [comment]
I didn't mean to close this @roj1512. The whole section about Deta needs to be updated. The links should be changed to https://deta.space and the information should be updated as well.

**Comment by roj1512** [comment]
So merge these as is and update the rest in another PQ or do everything in this PQ?

**Comment by dcdunkan** [comment]
> So merge these as is and update the rest in another PQ or do everything in this PQ?

That's for you guys to decide. As you like.

**Review by roj1512 — APPROVED** [review]

**Comment by roj1512** [comment]
I think you need to resolve the conflicts.

**Comment by Nazar-Ant** [comment]
conflicts are resolved

**Comment by roj1512** [comment]
Thanks!

File: site/docs/hosting/comparison.md [hunk 1]
```diff
@@ -33,21 +33,21 @@ Using [polling](../guide/deployment-types.md) is also possible with PaaS if the
...
+| Deta                   | Free       | No paid plans yet                                                                                          | No specific limits                                                                      | ✅                                                                                                      | ✅                                             | ✅                                                | Deno is supported by a [custom application](https://deta.space/docs/en/quickstart-guides/custom) ([example](https://github.com/deta/starters/tree/main/deno-app)). |
```
Comment (CikiMomogi) [inline site/docs/hosting/comparison.md:38]: 404
```suggestion
| Deta                   | Free       | No paid plans yet                                                                                          | No specific limits                                                                      | ✅                                                                                                      | ✅                                             | ✅                                                | Deno is supported by a [custom application](https://deta.space/docs/en/build/quick-starts/custom) ([example](https://github.com/deta/starters/tree/main/deno-app)). |
```

**Review by CikiMomogi — CHANGES_REQUESTED** [review]

**Review by roj1512 — APPROVED** [review]

**Review by LWJerri — APPROVED** [review]

**Review by habemuscode — APPROVED** [review]

**Review by CikiMomogi — APPROVED** [review]

**Pull request merged**